app/retail/templates/settings/account.html

```diff
@@ -70,11 +70,12 @@ <h5 class="font-subheader">{% trans "Delete My Account" %}</h5>
               '</a>' +
             '</div>' +
             '<div class="col-12 pt-2 pb-2 text-center">' +
-              '<h2 class="font-title">' + gettext('Delete Your Account') + '</h2>' +
+              '<h2 class="font-title">' + gettext('Are you sure you want to delete all of your data from Gitcoin?') + '</h2>' +
             '</div>' +
             '<div class="col-12 pt-2 pb-2 font-body">' +
               '<p>' + gettext('Once you delete your account, there is no going back.') + '<br>' + gettext('Please be certain.') + '</p>' +
               '<p>' + gettext('This action cannot be undone. This will permanently delete your account, stats, bounties funded and worked upon by you.') + '</p>' +
+              '<p>' + gettext('Your data however will not be removed from the ethereum blockchain.') + '</p>' +
```

Comment:
'Your data will not be removed from the Ethereum blockchain, which is immutable. However, this will no longer tie to your Gitcoin account.'

Reply:
What does tie to your Gitcoin account mean? Can we be more clear?

Let's use:
By clicking delete, all of your data will be removed from Gitcoin. We are unable to delete your data from the Ethereum blockchain.

cc:@vs77bb